packages/legend-query-builder/src/components/__tests__/QueryBuilderTDSWindowPanel.test.tsx

```diff
@@ -67,10 +159,8 @@ test(
       tdsState.setShowWindowFuncPanel(true);
     });
 
-    await waitFor(() =>
```

Comment:
I replaced all the instances of `await waitFor` with `await findBy` in this test file because it's cleaner, so that's why there's so many changes here.